test/lib/slots.test.ts

```diff
@@ -0,0 +1,56 @@
+import getSlots from '@lib/slots';
+import {it, expect} from '@jest/globals';
+import MockDate from 'mockdate';
+import dayjs, {Dayjs} from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+MockDate.set('2021-06-20T11:59:59Z');
+
+it('can fit 24 hourly slots for an empty day', async () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'can·fit·24·hourly·slots·for·an·empty·day'` with `"can·fit·24·hourly·slots·for·an·empty·day"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
it("can fit 24 hourly slots for an empty day", async () => {
```